vg/vgsvg/vgsvg.go

```diff
@@ -37,8 +39,10 @@ type Canvas struct {
 	svg  *svgo.SVG
 	w, h vg.Length
 
-	buf   *bytes.Buffer
-	stack []context
+	buf    *bytes.Buffer
+	stack  []context
+	styles map[string]string
+	svgID  string
```

Comment:
Why does this need to exist, it is never mutated after construction with `NewWith` as far as I can see.

Reply:
The idea was to define the svgID in one location, and I initially generated a random svgID. Now that it's a constant, I should indeed use a const string.